cypress/e2e/hermioneAccount.spec.cy.js

```diff
@@ -0,0 +1,44 @@
+/// <reference types='cypress' />
+
+describe('Bank app', () => {
+  before(() => {
+    cy.visit('/');
+  });
+
+  it('should provide the ability to work with Hermione\'s bank account', () => {
+    cy.contains('button', 'Customer Login').click();
+    cy.get('#userSelect').select('Hermoine Granger');
+    cy.get('.btn-default').should('be.visible').click();
+
+    cy.contains('.center', 'Account Number')
+      .should('contain.text', 1001);
```

Comment:
The account number is hardcoded as 1001. Ensure this value is consistent with the test data or consider parameterizing it if it might change.